peepingtom/core/alchemists/from_points.py

```diff
@@ -0,0 +1,37 @@
+import numpy as np
+
+from ..datablocks import LineBlock, ParticleBlock, PointBlock, OrientationBlock, PropertyBlock
+from ..base import Alchemist
+
+
+class PointToLineAlchemist(Alchemist):
+    """
+    transform a PointBlock into a LineBlock
+    """
+    def __init__(self, pointblock):
+        super().__init__({'points': pointblock})
+
+    def transform(self):
+        points = self.inputs['points'].data
+        if 'line' not in self.outputs:
+            self.outputs['line'] = LineBlock(points)
+        else:
+            self.outputs['line'].data = points
+
+
+class PointToParticleAlchemist(Alchemist):
```

Comment:
cool! Although particles should probably be initialised with random orientations